components/raftstore/src/store/worker/region.rs

```diff
@@ -896,8 +896,8 @@ mod tests {
         thread::sleep(Duration::from_millis(200));
         assert!(engine.kv.get_value(b"k1").unwrap().is_none());
         assert_eq!(engine.kv.get_value(b"k2").unwrap().unwrap(), b"v2");
-        for i in 0..9 {
-            assert!(engine.kv.get_value(&ranges[i]).unwrap().is_none());
+        for range in ranges.iter().take(9) {
```

Comment:
I don't feel it's a proper lint, it adds an assumption that `ranges` will always have a larger length.

Reply:
Agree. @dvermd This is a special case, so I think we can disable the lint here.

Reply:
The for on `0..9` has been restored.
If there's really a concern about the assumption about the size of `ranges`, shouldn't the function be reworked to define a variable of the upper bound of `0..10` and use it in all the `for` of this method.

```rust
let ranges_count = 10;

for i in 0..ranges_count {
    [...]
}

// and 
for i in 0..ranges_count - 1 {
    [...]
}
```

Reply:
Good idea!